src/sentry/tagstore/snuba/backend.py

```diff
@@ -412,6 +412,7 @@ def get_tag_keys(
             environment_id and [environment_id],
             denylist=denylist,
             tenant_ids=tenant_ids,
+            include_values_seen=include_values_seen,
```

Comment:
I'm not sure why this argument wasn't forwarded to `__get_tag_keys` in the first place. Anyways, it seems like the `get_tag_keys` wrapper isn't used anywhere except for the project `/tags` endpoint, so this should be safe.